cmd/micro/micro.go

```diff
@@ -521,7 +521,7 @@ func main() {
 					t.Resize()
 				}
 			case *tcell.EventMouse:
-				if !searching {
+				// if !searching {
```

Comment:
Commented out instead of deleting and unindenting, because it makes diff unintelligible unless you ignore whitespace.
Should probably be removed and unindented after pulling.